buf.gen.yaml

```diff
@@ -1,6 +1,7 @@
 version: v2
 plugins:
-  - remote: buf.build/protocolbuffers/go:v1.35.2
+  - remote: buf.build/protocolbuffers/go:v1.36.1
     out: private/gen/modules
     opt:
       - paths=source_relative
+      - default_api_level=API_HYBRID
```

Comment:
Don't think this is strictly necessary as other projects consume the gen SDK from this project, but figure we can start adding it to aid in any migrations.

Reply:
This is just to get the `state` generated code here https://github.com/bufbuild/modules/blob/6d35eac8b8212c4731c79833630fd85f605f4f43/private/gen/modules/state/v1alpha1/state.pb.go#L21. As you say, not strictly necessary, but having this `HYBRID` we can run `open2opaque` here too. Thanks!